push/templates/push/applications.html

```diff
@@ -117,7 +117,13 @@
         contentType: 'application/x-www-form-urlencoded; charset=UTF-8'
     });
 
-    api_call.always(function() {
+    api_call.fail(function(jqXHR) {
+      $.each(jqXHR.responseJSON, function(index, element) {
+        $('#id_' + index).after('<div class="alert callout">' + element + '</div>');
+      });
+    });
```

Comment:
This adds a `fail` handler to display the server-side validation errors below their corresponding `input` fields.